src/languages/fsharp.js

```diff
@@ -127,20 +123,47 @@ export default function(hljs) {
         ]
       },
       {
-        className: 'meta',
-        begin: '\\[<',
-        end: '>\\]',
-        relevance: 10
+        // computation expressions:
+        beginScope: "emphasis",
+        match: /\b[_a-z]\w*(?=\s*\{)/i
+      },
+      {
+        // preprocessor directives and fsi commands:
+        scope: 'meta',
+        begin: '#',
+        end: '$',
+        keywords: {
+          keyword: 'if else endif line nowarn light r i I load time help quit'
+        }
+      },
+      {
+        // [<Attributes("")>]
+        scope: 'meta',
+        begin: /^\s*\[<(?=[<\w])/,
+        excludeBegin: true,
+        end: />\]/,
+        excludeEnd: true,
+        relevance: 10,
+        contains: [
+          {
+            scope: 'string',
+            begin: /"/,
+            end: /"/
+          },
+          hljs.C_NUMBER_MODE
+        ]
       },
       {
-        className: 'symbol',
-        begin: '\\B(\'[A-Za-z])\\b',
-        contains: [hljs.BACKSLASH_ESCAPE]
+        scope: 'operator',
+        // only non arithmetic operators that we can confidently match:
+        match: /\|{1,3}>|<\|{1,3}|->|<-|(?<!<)<<(?!<)|(?<!>)>>(?!>)|:>|:\?>?|\.\.|::|(?<!\|)\|(?!\|)/
```

Comment:
Please use `regex.either` from utils and turn this into a nice easy to read array of operators, etc.